lib/utils/utils.dart

```diff
@@ -194,6 +195,20 @@ int formatIcon(String icon) {
   return int.parse('0x$icon');
 }
 
+Future<bool> checkGooglePlayServices() async {
+  if (Platform.isIOS) {
+    return true;
+  } else if (Platform.isAndroid) {
+    var availability = await GoogleApiAvailability.instance
+        .checkGooglePlayServicesAvailability();
```

Comment:
Is it possible for this to throw an exception? In the [example](https://pub.dev/packages/google_api_availability/versions) it looks like they are catching one.

Reply:
It appears that they are only catching the platform exception, which is only occurring for iOS devices. I have already added a check to deal with it, but I still wrapped my code inside a try/catch block.